app/actions/remote/scheduled_post.test.ts

```diff
@@ -97,7 +97,8 @@ beforeAll(() => {
 
 beforeEach(async () => {
     await DatabaseManager.init([serverUrl]);
-    operator = DatabaseManager.getServerDatabaseAndOperator(serverUrl)!.operator;
+    const serverDatabaseAndOperator = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+    operator = serverDatabaseAndOperator.operator;
```

Comment:
This change seems unneeded (just trying to keep the diff clean).

Reply:
Reverted.,